src/Pixidos/GPWebPay/Components/GPWebPayControl.php

```diff
@@ -62,8 +62,6 @@ class GPWebPayControl extends UI\Control
      */
     public function __construct(IOperation $operation, IProvider $provider, IContainer $control = null, $name = null)
     {
-        parent::__construct($control, $name);
```

Comment:
when you remove this, why not you does not remove `IContainer $control = null, $name = null` from constructor?